RangeImageHeader  Specifics
sensor_msgs/Image Image
